caster/lib/navigation.py

```diff
@@ -13,7 +13,8 @@
 from caster.lib import control
 
 
-DIRECTION_STANDARD={"sauce [E]": "up", "dunce [E]": "down", "lease [E]": "left", "Ross [E]": "right", "back": "left" }
+#DIRECTION_STANDARD={"sauce [E]": "up", "dunce [E]": "down", "lease [E]": "left", "Ross [E]": "right", "back": "left" }
+DIRECTION_STANDARD={"up": "up", "down": "down", "left": "left", "right": "right", "back": "left" }
```

Comment:
Incidentally, these change more than the "curse" command. They affect the direction key commands, ctrl/shift + direction keys, etc. See /lib/ccr/core/nav.py. 

Anyway, if you don't like any of the specs for the commands, beyond just changing them in your local copy of Caster, there is an update-proof mechanism for changing specs or actions in a given command at boot time without modifying anything which is tracked by git.

You are able to create "filter functions" in the user/ folder (which git ignores). When Caster boots and when you speak the commands to switch programming languages (or otherwise modify the active command set), Caster looks to see if any of these filter functions are present, and if so, modifies the command set however you specify before reactivating it. See user/filters/examples/scen4.py for some examples. Let me know if you have any questions.